airflow/providers/amazon/provider.yaml

```diff
@@ -576,3 +576,8 @@ secrets-backends:
 logging:
   - airflow.providers.amazon.aws.log.s3_task_handler.S3TaskHandler
   - airflow.providers.amazon.aws.log.cloudwatch_task_handler.CloudwatchTaskHandler
+
+additional-extras:
+  - name: cncf.kubernetes
+    dependencies:
+      - apache-airflow-providers-cncf-kubernetes>=5.0.0
```

Comment:
I thought this is automatically added as extra dependency, at least it listed in https://pypi.org/pypi/apache-airflow-providers-amazon/json or maybe we need specific version of provider?